fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb

```diff
@@ -16,6 +16,7 @@ def self.run(params)
 
         dsym_paths = []
         dsym_paths << params[:dsym_path] if params[:dsym_path]
+        dsym_paths += params[:dsym_paths] if params[:dsym_paths]
```

Comment:
Does it make sense to allow `:dsym_path` AND `:dsym_paths` ? I would think if we use one, we should disallow the other. Thoughts?

Reply:
@taquitos This _did_ cross my mind but `:dsym_path` can get autoset by some actions which might make things confusing by a user if also trying to use `:dsym_paths`. And I these parameters don't necessarily conflict - the `hockey` action has conflicting between `:ipa` and `:apk` because those both 100% can't be used at the same time where `:dsym_paths` more or less compliments/extends what `:dsym_path` can do 🙃

Reply:
I actually wanted to include the `:dsym_paths` functionality just inside of `:dsym_path` because they are super similar but the `:dsym_path` naming just didn't fit 😔 And I don't think we have the option in a `ConfigItem` to specif an "alias", do we???

Reply:
Ohhh good point that it can be autoset! It might be worthy of a code comment here to add that information, so future joshs don't try to remove this.